inventory/group_vars/.gitignore

```diff
@@ -0,0 +1 @@
+local_config_options.yml
```

Comment:
I think this is not needed?

Reply:
Ah, yes, I initially put the file in this dir. Just removed the .gitignore.